components/brave_player/browser/resources/playerEmbed.tsx

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import styled from 'styled-components'
+
+// For now we only supports youtube.
+// Usages: EmbedURL`youtube${videoId}`
+export function EmbedURL (source: string, videoId: string) {
+  if (source === 'youtube')
+    return `https://www.youtube-nocookie.com/embed/${videoId}?iv_load_policy=1&autoplay=1&rel=0&modestbranding=1`
+
+  return ''
+}
+
+const EmbedContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`
+
+const StyledFrame = styled.iframe`
+  width: 80%;
+  height: 80%;
+  border: none;
+`
+
+export function getFrameSrc (pathname: string) {
+  const paths = pathname.split('/')
+  if (paths.length < 3) {
+    return ''
+  }
+
+  const [, service, videoId] = paths
+
+  return EmbedURL(service, decodeURIComponent(videoId))
```

Comment:
We don't want to decode here. If it's a valid YT video ID - it can be used as is. If it has special symbols, such as `../`, we want to pass it encoded

Reply:
Aha I see. Thanks

Reply:
Aha, thanks. Removed.